pkgs/data/themes/adwaita-qt/default.nix

```diff
@@ -30,11 +31,18 @@ mkDerivation rec {
 
   buildInputs = [
     qtbase
-    qt5.qtx11extras
   ] ++ lib.optionals stdenv.isLinux [
     xorg.libxcb
+  ] ++ lib.optionals (!useQt6) [
+    qt5.qtx11extras
+  ] ++ lib.optionals useQt6 [
+    qtwayland
   ];
 
+  dontWrapQtApps = true;
```

Comment:
Also this

Reply:
I have to add one of them or the build failed. It's a requirement of `qt6Packages`.

Reply:
In that case I think
```nix
dontWrapQtApps = useQt6;
```
would be better.

Reply:
Done.